cmd/minishift/cmd/start.go

```diff
@@ -160,6 +160,15 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	if !isRestart {
+		// Run the default addons if MINISHIFT_HOME dir does not exist or empty
```

Comment:
@hferentschik , It is needed to delay the installation of addons of `minishift start` specific. Otherwise, for any command, the installation of default addons will occur.